.github/workflows/deploy-pages.yml

```diff
@@ -81,55 +39,18 @@ jobs:
       - name: Setup Pages
         id: pages
         uses: actions/configure-pages@v1
-      - name: Download Artifacts
-        uses: actions/download-artifact@v3
-        with:
-          path: artifacts
-      - name: Generate Data Aggregates
-        working-directory: ./
+      - name: Build
         run: |
           npm ci
-          mkdir ./munged
-
-          # download-artifact downloads artifacts in a directory named after the artifact
-          # details: https://github.com/actions/download-artifact#download-all-artifacts
-          for folder in ./artifacts/conformance-*.json; do
-            file="${folder}/output.json"
-            new_file="./munged/${folder#.\/artifacts\/conformance-}" # drop the ./artifacts/conformance- prefix
-            cat "$file" | node ./munge.js > "${new_file}"
-          done
-
-          # generate the sqlite database from our munged files
-          node ./munge_sql.js ./aggregates.db ./munged/*.json
+          make website
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OUTPUT_BASE_URL: ${{ steps.pages.outputs.base_url }}
       - name: Upload Data Aggregates
-        # will be very useful for local debugging
+        # Used for local debugging
         if: (failure() || success())
         uses: actions/upload-artifact@v3
         with:
-          name: dashboard-aggregates
+          name: artifacts
           path: |
-            ./munged
-            ./aggregates.db
-      - name: Generate Content
-        run: |
-          node ./munge_aggregates.js ./aggregates.db ./www
-      - name: Build with Hugo
-        run: |
-          hugo \
-            --minify \
-            --baseURL ${{ steps.pages.outputs.base_url }}
-        working-directory: www
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v1
-        with:
-          path: ./www/public
-  deploy:
-    environment:
-      name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
-    runs-on: ubuntu-latest
-    needs: build
-    steps:
-      - name: Deploy to GitHub Pages
-        id: deployment
-        uses: actions/deploy-pages@v1
+            ./artifacts
```

Comment:
I find this helpful for dealing with eof newlines.

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files#:~:text=Additionally%2C%20in%20the%20Settings%20page,there%20isn't%20already%20one.